nextflow.config

```
@@ -114,11 +114,11 @@ includeConfig 'conf/base.config'
 // Load modules.config for DSL2 module specific options
 includeConfig 'conf/modules.config'
 
```

We still need to load the normal nf-core custom profiles
```suggestion
// Load nf-core custom profiles from different Institutions
try {
    includeConfig "${params.custom_config_base}/nfcore_custom.config"
} catch (Exception e) {
    System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
}
```